packages/moda-sdk/src/profile/getName.test.ts

```diff
@@ -0,0 +1,66 @@
+import { getName } from "./getName";
+import { mockConfig } from "./testing/mockConfig";
+import { profileAddress } from "./testing/constants";
+import { AddressError, ConfigError } from "../types";
+
+describe('getName', () => {
+    it("Should return a list of names", async () => {
+        const result = await getName({
+            config: mockConfig,
+            profileAddress,
+        });
+
+        if (!result.ok) {
+            throw new Error("Expected result to be successful");
+        }
+
+        expect(result.ok).toBeTruthy;
```

Comment:
Tests like this line are redundant and can be dropped. Line 13 checks for this to be true.